kickstart/PXELinux_atomic.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+kind: PXELinux
+name: Atomic PXE
+description: Start Atomic from the bare metal (installer) image.
+oses:
+- CentOS-Atomic 7
+- Fedora-Atomic 22
+- RedHat-Atomic 7
+%>
+
+DEFAULT pxeboot
```

Comment:
I could add another case to the if statement that catches 'Atomic' on the name if that's what you're referring to? Otherwise I can't set the kernel or initrd instance variables without creating an specific Atomic OS family. It's almost identical to the default template for Fedora 16+ except for the `repo=` part and the paths for initrd/kernel.